client/app/components/faq/faqdetails.js

```diff
@@ -39,12 +37,12 @@ class FaqDetails extends React.Component {
 		const { state } = this.props.navigation;
 		return (
 			<ListSlider
-				renderItem={this.renderItem}
+				renderItem={this._renderItem}
```

Comment:
Probably should not be changed as fnc above still called `renderItem`

Reply:
Error due to merge conflict overwriting past changes.  Fixed, thanks.